hbase-server/src/main/resources/hbase-webapps/master/procedures.jsp

```diff
@@ -47,8 +47,28 @@
 <%@ page import="org.apache.hadoop.hbase.metrics.Histogram" %>
 <%@ page import="java.util.TreeMap" %>
 <%@ page import="org.apache.hadoop.hbase.metrics.impl.HistogramImpl" %>
+
+<jsp:include page="header.jsp">
```

Comment:
Why we move this page here? Because we may add some elements in the below code block and it should be placed after header?

Reply:
Yes, the below code block should be placed after header，as shown below.

![image](https://github.com/user-attachments/assets/cd0b46b2-7c6e-45e7-a5ed-58cca7d7a2dc)

If we don't do this, it will display as follows.

![image](https://github.com/user-attachments/assets/b145a971-2f61-4622-ba8f-29595ace60aa)